### PR TITLE
 chore: add installation tests

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

File: src/types/global.d.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright 2018 Google Inc. All Rights Reserved.
```
Comment (jinwoo) [inline src/types/global.d.ts]: Nit. Google LLC.

File: system-test/test-install.ts [hunk 2]
```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
```
Comment (jinwoo) [inline system-test/test-install.ts]: Same.

File: system-test/utils.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
```
Comment (jinwoo) [inline system-test/utils.ts]: Same.

File: src/types/global.d.ts (hunk 1 above)
Comment (DominicKramer) [reply src/types/global.d.ts]: Updated

File: system-test/test-install.ts (hunk 2 above)
Comment (DominicKramer) [reply system-test/test-install.ts]: Updated

File: system-test/utils.ts (hunk 3 above)
Comment (DominicKramer) [reply system-test/utils.ts]: Updated

**Pull request merged**